bundlex.exs

```diff
@@ -1,6 +1,25 @@
 defmodule Membrane.FFmpeg.SWResample.BundlexProject do
   use Bundlex.Project
 
+  defp get_ffmpeg() do
+    case Bundlex.get_target() do
+      %{os: "linux"} ->
+        {:precompiled,
+         "https://github.com/membraneframework-precompiled/precompiled_ffmpeg/releases/download/latest/ffmpeg_linux.tar.gz"}
```

Comment:
let's use updated implementation of `get_ffmpeg_url()`, like in other ffmpeg plugins